homeassistant/components/growatt_server/__init__.py

```diff
@@ -35,8 +37,7 @@ def get_device_list_classic(
     # Log in to api and fetch first plant if no plant id is defined.
     try:
         login_response = api.login(config[CONF_USERNAME], config[CONF_PASSWORD])
-        # DEBUG: Log the actual response structure
-    except Exception as ex:
+    except (requests.RequestException, json.JSONDecodeError) as ex:
```

Comment:
```suggestion
    except (RequestException, JSONDecodeError) as ex:
```

please only import the exceptions and not the entire module

But I'm wondering if 

```suggestion
    except GrowattError as ex:
```
wouldn't work as well, and that way we'd have a proper integration specific exception, which is nicer